samples/spring-webflux-security-xsuaa-usage/src/main/java/sample/spring/webflux/xsuaa/SecurityConfiguration.java

```diff
@@ -32,15 +31,13 @@ public class SecurityConfiguration {
 
 	private final Logger logger = LoggerFactory.getLogger(this.getClass());
 
-	@Autowired
```

Comment:
I think you can revert all changes in this file back and add the `xsuaa-spring-boot-starter` dependency in pom, which would clean up the pom nicely plus we could use the autoconfiguration capabilities.